src/Main.cpp

```diff
@@ -34,14 +34,16 @@ static bool hasFocus = true;
 static ScintillaGateway editor;
 static LPWORD kbBuff = (LPWORD)new byte[buffChars];
 static PBYTE kbState = new byte[256];
+TCHAR addChars[1024];
 
 static void enableSurroundSelection();
 static void showAbout();
 
 LRESULT CALLBACK KeyboardProc(int ncode, WPARAM wparam, LPARAM lparam);
 
 FuncItem funcItem[] = {
-	{ TEXT("Enable"), enableSurroundSelection, 0, false, nullptr },
+	// 2019-04-11:MVINCENT:Alt-key underline in menu
```

Comment:
The `2019-04-11:MVINCENT:` in these comments are not needed and can be removed.

Reply:
Agree, just providing context as to what I was doing, why adding the '&' character.  It underlines the following letter when the Alt key is pressed so keyboard shortcuts can be used without needing to assign a N++ shortcut menu command.

Reply:
The comment itself is fine. I just never see the need to add date/author information into code since that's the whole point of version control :)